src/services/services.ts

```diff
@@ -3798,6 +3798,10 @@ module ts {
                 // Try getting just type at this position and show
                 switch (node.kind) {
                     case SyntaxKind.Identifier:
+                        if(isLabelName(node)){

```

Comment:
all label checks are in the services layer. Let's keep it this way. the checker functions do not expect calls to labels. so @GuillaumeSalles did the right thing here.

I agree with @CyrusNajmabadi that labels should have symbols, and should be tracked by the checker. This way the services layer does not have to worry about it, and we remove all isLable checks from different LS functions; but that is a different change.